### ISSUE TITLE
Bug [DEV CHANNEL SPECIFIC]: ModernFlyouts does not replace the new W11 flyout

### ISSUE THREAD

**Issue by Rexogamer** [ModernFlyouts-Community/ModernFlyouts#920]
### ModernFlyouts Version

0.9.3.0

### Windows version

Windows 11 Insider Preview 

### OS build

22533.1001

### Platform

x64

### Area(s) with issue?

General

### Steps to reproduce

- Make sure you're in the Dev ring (at the time of writing this)
- Update to the latest version of W11 (at the time of writing this, 22533.1001)
- If it isn't configured to start on startup, run ModernFlyouts
- Trigger any action that would trigger both the built-in flyout and ModernFlyous (e.g. change your volume or brightness)

### ✔️ Expected Behavior

The ModernFlyouts flyout replaces the built-in one.

### ❌ Actual Behavior

Both flyouts display:
![image](https://user-images.githubusercontent.com/42586271/149222869-e2ff39b2-4793-4f59-ab80-13ccb7f1660e.png)

### Additional Context

I suspect that due to the completely new built-in flyout something (file locations?) that MF currently relies on/overwrites is no longer used?

**Comment by walkxcode** [comment]
Same for Lock Keys.

**Comment by GreepTheSheep** [comment]
Same here, build 22538
![image](https://user-images.githubusercontent.com/42576124/150290535-9e7c0f6a-bd75-459a-8c98-3f6f981ce22c.png)

**Comment by KeremTheDeveloper** [comment]
Same here too.

**Comment by MaxALEau** [comment]
Same

**Comment by leo283** [comment]
Yep, same here

**Comment by FadeMind** [comment]
Same Beta/Dev
![20220324-1648122424](https://user-images.githubusercontent.com/9846948/159909912-1c6e4fbf-451e-44a9-be0d-106a9cf5a6cf.png)

![20220324-1648122071](https://user-images.githubusercontent.com/9846948/159910094-ff4b8aa5-16da-49a8-ab8e-46b98fa9feb6.png)

**Comment by FaceHiddenInsideTheDark** [comment]
Same here, this is probably related to the new rework of the flyout interface. Hopefully fixing this doesn't break support for older versions.

**Comment by DeZenTID** [comment]
This happens on the beta branch too now since both branch merged on this update

**Comment by TPAXcc** [comment]
> 这里也一样，这可能与弹出界面的新返工有关。 希望修复此问题不会破坏对旧版本 Windows 的支持。

Same too.

**Comment by Samuel12321** [comment]
We are working on this for 0.11

**Comment by yodaluca23** [comment]
Yup this is happening to me too in the new 22H2, just updated...

**Comment by PeterNjeim** [comment]
> Yup this is happening to me too in the new 22H2, just updated...

@yodaluca23 make sure to read the whole thread before replying. They're working on it for version `0.11`.

**Comment by yodaluca23** [comment]
> > Yup this is happening to me too in the new 22H2, just updated...
> 
> @yodaluca23 make sure to read the whole thread before replying. They're working on it for version 0.11`.

I saw, just didn't see anyone specifying that it did not work in the non-beta's... so I figured I'd reply...

**Comment by PeterNjeim** [comment]
> I saw, just didn't see anyone specifying that it did not work in the non-beta's... so I figured I'd reply...

Makes sense. For your information the betas have very little difference from the final release. Just minor changes. For example, when I was using Windows 11 Beta before its initial release, the last beta from September 23rd ended up being the same exact build as the official release on October 5th.

Also a question for you, why do you put ellipses at the end of your comments? It could be misinterpreted as some sort of passive aggressive thing, though I don't think that's why you're using it

**Comment by ZhangDigital** [comment]
#1115 Same

**Comment by foursplaysroblox** [comment]
same here

**Comment by Ridoou** [comment]
I use HideVolumeOSD by Marcus Venturi to hide the W11 one

**Comment by bigplayer-ai** [comment]
same here

**Comment by PeterNjeim** [comment]
Since there's a lot of unnecessary spam in this thread from users who don't understand that the problem has already been acknowledged, to those who don't want their email bombarded, update your subscription settings for this issue to only notify you when the issue has been closed (click `Customize`):
![image](https://user-images.githubusercontent.com/26418974/211435351-d2834360-2cfd-4f2c-9483-9e519ac8277d.png)
![image](https://user-images.githubusercontent.com/26418974/211435347-a19c6939-1527-4f97-8d1e-a44dd516dc11.png)

**Comment by jxolotl** [comment]
This [fork version of HideVolumeOSD](https://github.com/gpkgpk/HideVolumeOSD/releases) seems to work more reliably than the [original version](http://wordpress.venturi.de/?p=379) for me on Windows 11, resuming from sleep doesn't reactivate windows 11 volume osd.

**Comment by DeZenTID** [comment]
still happening, but this could be a temporary fix 

> I use HideVolumeOSD by Marcus Venturi to hide the W11 one

**Comment by eshuaiii** [comment]
Another thought - I don't know how difficult it is to implement, but if removing the native Windows flyout is difficult, would having an option to just disable the ModernFlyout's volume slider but show any playback media be a temporary workaround?

**Comment by Atm2222** [comment]
> 

Yes this is exactly what I would like to be able to do too. Even if the ability to hide the new flyouts gets added, I would love if I could use the new windows volume flyouts but keep the media flyout that would only open when I pause/play the music

**Comment by thedtvn** [comment]
> still happening, but this could be a temporary fix
> 
> > I use HideVolumeOSD by Marcus Venturi to hide the W11 one
@DeZenTID I think not I am use windown  11 beta and it work very well

**Comment by ckaspers** [comment]
Considering the fact that the last update to this app was a year ago and there are over 200 open issues, I have to question whether any development is even being performed. I think this app has become abandonware.

**Comment by vxsqi** [comment]
Same

**Comment by manstheram** [comment]
I've submitted a pull-request that fixes this, so hopefully it's added soon.

**Comment by manstheram** [comment]
It has been fixed in v10 beta 3.

**Comment by ckaspers** [comment]
Can confirm it is indeed fixed in the beta, thanks for your effort!

**Comment by Ridoou** [comment]
Time to use this software again 😄

**Comment by guguss31** [comment]
Pareil depuis que j'ai installé Windows 11 .

**Comment by vvirtues** [comment]
still a problem

**Comment by vvipallwe** [comment]
![Uploading image.png…]()
same
